mpls/draft-ietf-detnet-mpls.xml

```diff
@@ -1022,38 +1022,43 @@ DFx = DetNet member flow x over a TE LSP
        DetNet-aware forwarder does duplicate frame elimination based on
        the flow identification and the sequence number combination. The
        packet replication is also done within the DetNet-aware
-       forwarder. During elimination and the replication process the
-       sequence number of the DetNet member flow MUST be preserved and
-       copied to the egress DetNet member flow.
-     </t>
-     <t>
-       The internal design of a relay node is out of scope of this
-       document. However the reader's attention is drawn to the need to
-       make any PREOF state available to the packet processor(s) dealing
-       with packets to which the PREOF functions must be applied, and to
-       maintain that state is such as way that it is available to the
-       packet processor operation on the next packet in the DetNet flow
-       (which may be a duplicate, a late packet, or the next packet in
-       sequence.
+       forwarder. 
      </t>
    </section>
 
 
    <section title="Relay Node Processing" anchor="sec_s_pe">
      <t>
-       A DetNet Relay node operates in the DetNet forwarding sub-layer .  For
-       DetNet using MPLS this processing is performed on the F-Label.  This
+       A DetNet Relay node operates in the DetNet forwarding sub-layer and service sub-layer.  For
+       DetNet using MPLS forwarding processing is performed on the F-Label.  This
        processing is done within an extended forwarder function. Whether an
        ingress DetNet member flow receives DetNet specific processing depends
        on how the forwarding is programmed.  Some relay nodes may be DetNet
-       service aware, while others may be unmodified LSRs that only understand
-       how to switch MPLS-TE LSPs.
-     </t>
-     <t>
-       It is also possible to treat the relay node as a transit node,
+       service aware for certain DetNet services, while for other DetNet services 
+	   the node may perform as unmodified LSRs that only understand

```

Comment:
these nodes may perform

(Plural because the sentence begins with a plural. Alternatively talks about a singular node in the whole sentence.)